check.py

```diff
@@ -123,6 +123,13 @@ def setup_waterfall():
 
 # utilities
 
+def run_command(cmd, stderr=None):
+  print 'executing: ', ' '.join(cmd)
+  proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=stderr)
```

Comment:
This could just be 
`subprocess.check_output(cmd, stdout=subprocess.PIPE)`
since you assert that the return code is 0